nx_cugraph/algorithms/bipartite/centrality.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cupy as cp
+import pylibcugraph as plc
+
+from nx_cugraph.convert import _to_graph
+from nx_cugraph.utils import networkx_algorithm
+
+__all__ = ["betweenness_centrality"]
+
+
+@networkx_algorithm(
+    name="bipartite_betweenness_centrality",
+    version_added="24.12",
+    _plc="betweenness_centrality",
+)
+def betweenness_centrality(G, nodes):
+    G = _to_graph(G)
+
+    node_ids, values = plc.betweenness_centrality(
+        resource_handle=plc.ResourceHandle(),
+        graph=G._get_plc_graph(),
+        k=None,
+        random_state=None,
+        normalized=False,
+        include_endpoints=False,
+        do_expensive_check=False,
+    )
+    top_node_ids = G._nodekeys_to_nodearray(set(nodes))
+    bottom_node_ids = cp.setdiff1d(
```

Comment:
While pairing, @nv-rliu suggested we could maybe use `cp.delete`. I think we should b/c it's faster.
```suggestion
    bottom_node_ids = cp.delete(
```
Here are some microbenchmarks that compare them:
```python
In [1]: import cupy as cp

In [2]: a = cp.arange(10_000_000)

In [3]: b = cp.unique(cp.random.random_integers(a.size-1, size=a.size*3//4))

In [4]: b.size
Out[4]: 5278367

In [5]: assert (cp.delete(a, b) == cp.setdiff1d(a, b, assume_unique=True)).all()

In [6]: %timeit cp.setdiff1d(a, b, assume_unique=True)
2.52 ms ± 349 μs per loop (mean ± std. dev. of 7 runs, 10 loops each)

In [7]: %timeit cp.delete(a, b)
620 μs ± 6.65 μs per loop (mean ± std. dev. of 7 runs, 100 loops each)

In [8]: cp.random.shuffle(b)

In [9]: %timeit cp.delete(a, b)
1.16 ms ± 3.65 μs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)

In [10]: %timeit cp.setdiff1d(a, b)
7.76 ms ± 18.9 μs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```